File: server/groute/src/main/java/com/ssafy/groute/dto/board/BoardDetail.java
package com.ssafy.groute.dto.board;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class BoardDetail {
    private int id;
    private String title;
    private String content;
    private String img;
    private String createDate;
    private String updateDate;
    private int heartCnt;
    private int hitCnt;
    private int boardId;
    private String userId;
    private int placeId;
}

File: client/Groute/app/src/main/java/com/ssafy/groute/src/dto/BoardDetail.kt
package com.ssafy.groute.src.dto

data class BoardDetail(
    val id: Int,
    var title: String,
    var content: String,
    var img: String,
    val createDate: String,
    val updateDate: String,
    var heartCnt: Int,
    val hitCnt: Int,
    val boardId: Int,
    val userId: String,
    val placeId: Int,
    val commentCnt : Int
){
    constructor(title:String,content:String,img:String, boardId:Int, userId:String):this(0,title,content,img,"","",0,0, boardId,userId,0,0)
    constructor(id:Int,title:String,content:String,img:String, boardId:Int, userId:String): this(id,title,content,img,"","",0,0, boardId,userId,0,0)
    constructor(title:String,content:String,img:String, boardId:Int, userId:String, placeId:Int):this(0,title,content,img,"","",0,0, boardId,userId,placeId,0)
    constructor(id:Int, title:String,content:String,img:String, boardId:Int, userId:String, placeId:Int):this(id,title,content,img,"","",0,0, boardId,userId,placeId,0)
    constructor(title:String,content:String,img:String, boardId:Int, heartCnt: Int, userId:String):this(0,title,content,img,"","",heartCnt,0, boardId,userId,0,0)
}

